docs/en/index.rst

```diff
@@ -59,7 +63,10 @@ Example of configuring the authentication middleware using ``authentication`` ap
             $service->loadAuthenticator('Authentication.Session');
             $service->loadAuthenticator('Authentication.Form', [
                 'fields' => $fields,
-                'loginUrl' => '/users/login'
+                'loginUrl' => Router::url([
+                                    'controller' => 'Users',
+                                    'action' => 'login'
+                                    ])
```

Comment:
```suggestion
                'loginUrl' => Router::url([
                    'prefix' => false,
                    'plugin' => null,
                    'controller' => 'Users',
                    'action' => 'login',
                ])
```

Reply:
Same would apply above, if using the router here at all, then it should also set the `prefix` and `plugin` keys in order to avoid missing route errors when navigating in prefix/plugin scopes.

I'm wondering if it's still necessary to "unset" `prefix` and `plugin` differently, ie `false` vs `null`.

Reply:
> I'm wondering if it's still necessary to "unset" prefix and plugin differently, ie false vs null.

It is still important to set them to null or false.